build/mac/BUILD.gn

```diff
@@ -242,16 +254,111 @@ if (skip_signing) {
       shell_script,
       unsigned_dmg,
     ]
-    outputs = [ "${root_out_dir}/$mac_signing_output_prefix/$brave_dmg" ]
+    outputs = [ "${root_out_dir}/$mac_signed_output_prefix/$brave_dmg" ]
     args = [
       rebase_path(shell_script, root_build_dir),
       rebase_path(unsigned_dmg),
-      rebase_path("$root_out_dir/$brave_dmg"),
+      rebase_path("$root_out_dir/$mac_signed_output_prefix/$brave_dmg"),
       keychain_db,
       mac_signing_identifier,
       "-r=designated => identifier \"$chrome_product_full_name\" and certificate leaf = H\"$mac_signing_identifier\"",
     ]
 
     deps = [":create_dmg"]
   }
+
+  group("finalize_dmg_pkg") {
+    deps = [
+      ":copy_dmg_pkg"
+    ]
+  }
+
+  copy("copy_notarize_script") {
```

Comment:
why are we copying the script?

Reply:
copying the file doesn't change where it runs from and it shouldn't be in build/mac in the first place. The scripts go in brave/scripts and they work just fine there

Reply:
I guess chrome also does this for some reason so don't worry about it for now